Assets/Helpers/bevy_turborand.toml

```
@@ -0,0 +1,3 @@
+name = "bevy_turborand"
+description = "a fast random number generator based on turborand"
```

```suggestion
description = "ECS enabled random number generation based on turborand"
```

Maybe the better phrased description? Best to summarise why such a plugin exists, instead of just using `turborand` or `rand` et al directly. Thoughts?

---

Agreed, this is clearer. I would do ECS integrated instead?

---

`ECS integrated` is better, yes, I'd go for that as well.